psi4/src/psi4/lib3index/dfhelper.cc

```diff
@@ -2942,7 +3054,14 @@ void DFHelper::compute_JK(std::vector<SharedMatrix> Cleft, std::vector<SharedMat
         M = std::unique_ptr<double[]>(new double[tots]);
         Mp = M.get();
     } else
-        Mp = Ppq_.get();
+        if (!wcombine_) Mp = Ppq_.get();
+
+    double* M1p;
+    if (wcombine_) {
+        M1p = m1Ppq_.get();
+    } else {
+        //M1p = nullptr;
```

Comment:
what's supposed to be happening in this `else {}`?

some safety brackets around the `} else` above probably advisable.

maybe evaluate whether the added/removed newlines were intentional.

Reply:
Agreed. Removal is probably better.